vm/runtime.ts

```diff
@@ -666,6 +666,9 @@ module J2ME {
     TWO_PWR_32_DBL = 4294967296,
     TWO_PWR_63_DBL = 9223372036854776000,
 
+    // The target amount of free memory before garbage collection is forced on unwind.
+    FREE_MEMORY_TARGET = 4086,
```

Comment:
We should probably set it to a lower value, otherwise we'll run collections too rarely.